crates/napi/src/bindgen_runtime/js_values/serde.rs

```diff
@@ -111,14 +113,33 @@ impl FromNapiValue for Map<String, Value> {
 
 impl ToNapiValue for Number {
   unsafe fn to_napi_value(env: sys::napi_env, n: Self) -> Result<sys::napi_value> {
+    #[cfg(feature = "napi6")]
+    let max_safe_int = (2f64.powi(53) - 1f64) as i64;
```

Comment:
```suggestion
    const MAX_SAFE_INT: i64 = (2f64.powi(53) - 1f64) as i64;
```

Reply:
This produces an error: cannot call non-const fn `std::f64::<impl f64>::powi` in constants calls in constants are limited to constant functions, tuple structs and tuple variants

I will substitute to the constant number and comment the expression